apps/meteor/client/providers/CallProvider/CallProvider.tsx

```diff
@@ -161,20 +163,36 @@ export const CallProvider: FC = ({ children }) => {
 		roomCoordinator.openRouteLink('v', { rid });
 	}, []);
 
+	const findOrCreateVisitor = useCallback(
+		async (caller: ICallerInfo): Promise<ILivechatVisitor> => {
+			const phone = parseOutboundPhoneNumber(caller.callerId);
+
+			const { contact } = await getContactBy({ phone });
+
+			if (contact) {
+				return contact as unknown as ILivechatVisitor;
```

Comment:
The type returned by `useEndpoint` is not compatible with the same type on `core-typings`. If any of you know how to solve this, please let me know.

Reply:
Yes, this is because of Types Serialization, when we send some data over HTTP requests, the objects and types are converted to strings, in this case `Date` type is converted to `string` type which creates this problem.